content/en/blog/_posts/2022-TBD-storage-capacity-GA/index.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: blog
+title: "Storage Capacity Tracking reaches GA in Kubernetes 1.24"
+date: 2022-12-31
+slug: storage-capacity-ga
+---
+
+ **Authors:** Patrick Ohly (Intel)
+
+The v1.24 release of Kubernetes brings _storage capacity tracking_ as a generally available feature.
+
+## Problems we are solving
+
+As explained in more detail in the [previous blog post about this
+feature](/blog/2021/04/14/local-storage-features-go-beta/), storage capacity
+tracking allows a CSI driver to publish information about remaining
+capacity. The kube-scheduler then uses that information to pick suitable nodes
+for a Pod when that Pod has volumes that still need to be provisioned.
+
+Without this information, a Pod may get stuck without ever being scheduled onto
+a suitable nodes because kube-scheduler has to choose blindly and always ends
+up picking a node for which the volume cannot be provisioned because the
+CSI driver does not have sufficient storage left.
+
+Because CSI drivers publish storage capacity information that gets used at a
+later time when it might not be up-to-date anymore, it can still happen that a
+node is picked that doesn't work out after all. Volume provisioning recovers
+from that by informing the scheduler that it needs to try again with a
+different node.
+
+[Load
+tests](https://github.com/kubernetes-csi/csi-driver-host-path/blob/master/docs/storage-capacity-tracking.md)
+that were done again for promotion to GA confirmed that all storage in a
+cluster can be consumed by Pods with storage capacity tracking whereas Pods got
+stuck without it.
+
+## Problems we are *not* solving
```

Comment:
Problems we have *not* solved

Reply:
That's indeed better because it doesn't rule out that we'll solve them in the future.

But then we also need to say "Problems we have solved", for consistency and to avoid the impression that we are still working on those, right?

I changed both.